docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
 
   gstreamer-video:
     image: ivadim/fruitnanny-gstreamer
-    restart: always
+    restart: "no"
```

Comment:
What is the reason to set restart to "no"?

Reply:
It's actually a mistake, sorry.
I changed it for my tests because I didn't want containers to be restarted after a RPi reboot.